Sources/StreamChatUI/ChatChannel/ChatChannelVC.swift

```diff
@@ -180,28 +230,19 @@ open class ChatChannelVC: _ViewController,
         )
     }
 
+    open func chatMessageListVCShouldLoadFirstPage(
```

Comment:
Bit of explanation here please.

Reply:
The explanation is available on the delegate `ChatMessageListVCDelegate`